src/test/resources/alma-fix/99374868243506441.json

```diff
@@ -0,0 +1,394 @@
+  "almaMmsId" : "99374868243506441",
+  "hbzId" : "HT030866325",
+  "deprecatedUri" : "http://lobid.org/resources/HT030866325#!",
+  "isbn" : [ "9783826423475", "382642347X" ],
+  "oclcNumber" : [ "1459708359" ],
+  "dnbId" : "1312088311",
+  "alternativeTitle" : [ "ADAC Mosel, Hunsrück, Eifel & Pfälzerwald", "Mosel, Hunsrück, Eifel ..." ],
+  "otherTitleInformation" : [ "Regionalkarte : aus einer anderen Perspektive die Umgebung erleben" ],
+  "edition" : [ "2. [Auflage], [Laufzeit]  2028" ],
+  "publication" : [ {
+    "startDate" : "2024",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "Ostfildern" ],
+    "publishedBy" : [ "MairDumont GmbH & Co. KG" ]
+  } ],
+  "describedBy" : {
+    "id" : "http://lobid.org/resources/99374868243506441",
+    "label" : "Webseite der hbz-Ressource 99374868243506441",
+    "type" : [ "BibliographicDescription" ],
+    "inDataset" : {
+      "id" : "http://lobid.org/resources/dataset#!",
+      "label" : "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data"
+    },
+    "resultOf" : {
+      "type" : [ "CreateAction" ],
+      "endTime" : "0000-00-00T00:00:00",
+      "instrument" : {
+        "id" : "https://github.com/hbz/lobid-resources",
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "https://lobid.org/marcxml/99374868243506441",
+        "dateCreated" : "2024-10-14",
+        "dateModified" : "2025-01-21",
+        "type" : [ "DataFeedItem" ],
+        "label" : "hbz-Ressource 99374868243506441 im Exportformat MARC21 XML",
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        },
+        "sourceOrganization" : {
+          "id" : "http://lobid.org/organisations/DE-6#!",
+          "label" : "Universitäts- und Landesbibliothek Münster, Zentralbibliothek"
+        },
+        "provider" : {
+          "id" : "http://lobid.org/organisations/DE-6#!",
+          "label" : "Universitäts- und Landesbibliothek Münster, Zentralbibliothek"
+        },
+        "modifiedBy" : [ {
+          "id" : "http://lobid.org/organisations/DE-6#!",
+          "label" : "Universitäts- und Landesbibliothek Münster, Zentralbibliothek"
+        } ]
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ]
+  },
+  "sameAs" : [ {
+    "id" : "https://hub.culturegraph.org/resource/(DE-605)99374868243506441",
+    "label" : "Culturegraph Ressource"
+  }, {
+    "id" : "http://worldcat.org/oclc/1459708359",
+    "label" : "OCLC Ressource"
+  }, {
+    "id" : "https://d-nb.info/1312088311",
+    "label" : "DNB-Ressource"
+  }, {
+    "id" : "http://nwbib.de/99374868243506441#!",
+    "label" : "NWBib-Ressource"
+  } ],
+  "isPartOf" : [ {
+    "type" : [ "IsPartOfRelation" ],
+    "numbering" : "11"
+  } ],
```

Comment:
I am nit sure I understand this. The file is invalid against the updated schema. Is this somehow taken into account in any tests?